.github/workflows/ci.yml

```diff
@@ -115,3 +115,32 @@ jobs:
 
       - name: Run unit tests
         run: vendor/bin/phpunit
+
+  mutation-tests:
+    name: "Mutation Tests"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.0
+          coverage: xdebug
+          tools: composer
+
+      - name: Get composer cache directory
+        id: composer-cache
+        run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+
+      - name: Cache dependencies
+        uses: actions/cache@v3.3.1
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: ${{ runner.os }}-composer-
+
+      - name: Install dependencies
+        run: composer install --no-interaction --no-ansi --no-progress
+
+      - name: Run mutation tests
+        run: ./vendor/bin/infection  --show-mutations --threads=$(nproc) --min-msi=100 --min-covered-msi=100
```

Comment:
You can use `--threads=max` that will automatically get the number of cores on any OS

Reply:
Thank you very much @maks-rafalko! The truth is that we discovered it yesterday while we were adding it to Gacela too.

Reply:
Thanks, @maks-rafalko. I already changed it here https://github.com/gacela-project/router/commit/84df2cc5be4bb6e2ed8ebfa89e006a631cba931c